sockeye/encoder.py

```diff
@@ -226,6 +265,24 @@ def get_rnn_cells(self) -> List[mx.rnn.BaseRNNCell]:
                 cells.append(cell)
         return cells
 
+    def get_encoded_data_length(self, data_length: mx.sym.Symbol) -> mx.sym.Symbol:
+        """
+        Returns the data lengths of the encoded sequence.
+        """
+        current_data_length = data_length
```

Comment:
Nevermind, got it. But if we let encode() return this information directly it might be easier for callers